lib/new_relic/agent/instrumentation/grpc/client/instrumentation.rb

```diff
@@ -73,10 +70,16 @@ def trace_with_newrelic?(host = nil)
             return do_trace unless do_trace.nil?
 
             host ||= @host
-            return false unless host && !host.match?(HOST_BLOCKLIST_PATTERN)
+            return false unless host && !host_denylisted(host)
 
             true
           end
+
+          def host_denylisted(host)
```

Comment:
Since we're aiming to return a boolean here, would you mind adding a question mark to the end of the method name?

Reply:
resolved: 4040d521ecd26241c7c92c96cfa89d96cc6ed147